packages/create/src/recipes/nuxt/index.ts

```diff
@@ -0,0 +1,50 @@
+import path from "node:path";
+import debug from "debug";
+import { updatePackage } from "write-package";
+
+import type { BaseOptions, Recipe } from "../types.js";
+import { copyTemplateFiles } from "../../utils.js";
+
+const logger = debug("@edgedb/create:recipe:nuxt");
+
+// interface NuxtOptions {}
+
+const recipe: Recipe = {
+  skip(opts: BaseOptions) {
+    return opts.framework !== "nuxt";
+  },
+  async apply(
+    { projectDir }: BaseOptions,
+  ) {
+    logger("Running nuxt recipe");
+
+    const dirname = path.dirname(new URL(import.meta.url).pathname);
+
+    await copyTemplateFiles(
+      path.resolve(dirname, 'template'),
+      projectDir
+    );
+
+    await updatePackage(projectDir, {
+      type: 'module',
+      scripts: {
+        dev: "nuxi dev",
+        build: "nuxi build",
+        start: "nuxi start",
+        generate: "nuxi generate"
+      },
+      dependencies: {
+        "@iconify-json/heroicons": "1.1.19",
+        "@nuxt/ui": "^2.13.0"
+      },
+      devDependencies: {
+        "nuxt-edgedb-module": "latest",
```

Comment:
I imagine this is a runtime dependency not just a dev dependency, right? Also, can we use a more strict version range? Maybe `^0.0.42`?

Reply:
Sure, it has runtime as well.